wgsl/index.bs

```diff
@@ -368,38 +368,43 @@ However, UTF-8 is always a valid encoding for a WGSL program.
 Note: The intent of promoting UTF-8 like this is to simplify interchange of WGSL programs
 and to encourage interoperability among tools.
 
-WGSL program text consists of a sequence of characters, grouped into contiguous non-empty sets forming:
+WGSL program text consists of a sequence of code points, grouped into contiguous non-empty sets forming:
 * [=comments=]
 * [=tokens=]
 * [=blankspace=]
 
-The program text must not include a null character.
+The program text must not include a null code point (`U+0000`).
 
-<dfn>Blankspace</dfn> is any combination of one or more of the following characters:
-* space
-* horizontal tab
-* linefeed
-* vertical tab
-* formfeed
-* carriage return
+<dfn>Blankspace</dfn> is any combination of one or more of the following code points:
+* space (`U+0020`)
+* horizontal tab (`U+0009`)
+* line feed (`U+000A`)
+* vertical tab (`U+000B`)
+* form feed (`U+000C`)
+* carriage return (`U+000D`)
+* next line (`U+0085`)
+* left-to-right mark (`U+200E`)
+* right-to-left mark (`U+200F`)
+* line separator (`U+2028`)
+* paragraph separator (`U+2029`)
 
 <div class='syntax' noexport='true'>
-  <dfn for=syntax>_space</dfn> :
+  <dfn for=syntax>_blankspace</dfn> :
 
-    | `/\s/`
+    | `/[\u0020\u0009\u000a\u000b\u000c\u000d\u0085\u200e\u200f\u2028\u2029]/uy`
 </div>
 
 To parse a WGSL program:
 1. Remove comments:
-    * Replace the first comment with a space character.
+    * Replace the first comment with a space code point.
```

Comment:
Should this be explicit as in other cases?

```suggestion
    * Replace the first comment with a space code point (`U+0020`).
```

Reply:
@dj2 Good suggestion, applied, thanks!